tests/integration/test_api/test_config/test_logs/test_logs.py

```diff
@@ -47,9 +47,12 @@
 tags:
```

Comment:
It should be desirable to add the ids for each tests case.
```
api_configurations = get_api_conf(configurations_path)

cases_ids = [configuration['configuration']['logs']['level'] for configuration in api_configurations]

@pytest.fixture(scope='module', params=api_configurations, ids=cases_ids)
def get_configuration(request):
```

Reply:
Done